### PR TITLE
TOMEE-2822-Translate to Portuguese: examples/jsonb-configuration

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira : https://issues.apache.org/jira/browse/TOMEE-2822

### CODE REVIEW

File: examples/jsonb-configuration/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+:jbake-status: status=published
+= Configuração JSON-B
+
+Este exemplo mostra como customizar jsonb para uma aplicaçao JAX-RS.
+JSONB é o novo padrão javaee-api: 8.0 para json serialização/desserialização. 
+São necessárias poucas anotações e o JsonbConfig oferece muitas configurações.
+
+== Executando e testando o Endpoint
+
+o aplicativo pode ser executado com `mvn clean install tomee:run' se porta
```
Comment (hbelmiro) [inline examples/jsonb-configuration/README_pt.adoc:12]:
```suggestion
o aplicativo pode ser executado com `mvn clean install tomee:run' se a porta
```

Comment (hbelmiro) [inline examples/jsonb-configuration/README_pt.adoc:12]:
```suggestion
o aplicativo pode ser executado com `mvn clean install tomee:run' se a porta
```

File: examples/jsonb-configuration/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    public Jsonb getContext(Class<?> type) {
+        return jsonb;
+    }
+
+}
+....
+
+== Acessando o endpoint rest
+
+O teste gera um aplicativo da web openejb e chama o terminal do usuário
```
Comment (hbelmiro) [inline examples/jsonb-configuration/README_pt.adoc:117]:
```suggestion
O teste gera uma aplicação web openejb e chama o endpoint do usuário
```

File: examples/jsonb-configuration/README_pt.adoc [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+
+Results :
+
+Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
+    
+....
+
+=== Dentro do jar
+
+javaee-api:8.0 traz todas as dependências necessárias para girar
```
Comment (hbelmiro) [inline examples/jsonb-configuration/README_pt.adoc:267]:
```suggestion
javaee-api:8.0 traz todas as dependências necessárias para iniciar
```

**Comment by Daniel-Dos** [comment]
thanks @hbelmiro for review .

**Pull request merged**